src/plugins/janus_streaming.c

```diff
@@ -4226,6 +4239,14 @@ static json_t *janus_streaming_process_synchronous_request(janus_streaming_sessi
 			char *old_metadata = mp->metadata;
 			char *new_metadata = g_strdup(json_string_value(md));
 			mp->metadata = new_metadata;
+			if (send_edited_event == TRUE) {
```

Comment:
Code style not: please remove the space between `if` and bracket (same for other occurrences below).

Reply:
Should be ok now.